Autopilot/Inc/airspeed.hpp

```diff
@@ -55,6 +58,58 @@ class airspeed {
  * Derived classes
  **********************************************************************************************************************/
 
+class MPXV7002DP : public airspeed {
+    public:
+        MPXV7002DP(const MPXV7002DP*) = delete;
+        static MPXV7002DP* GetInstance();
+        /**GetResult should:
+         * Moves values from the variables to the airspeed data struct.
+         * Sets the dataNew variable to false after that.
+         */
+        void GetResult(airspeedData_t &Data);
+
+        /**
+         *  Triggers interrupt for new airspeed measurement - stores 
+         *  raw data in variables. This function is blocking, as polling for the adc has
+         *  a timeout of 50 ms.
+         */
```

Comment:
If this is blocking our entire program, 50 ms may be a bit concerning since this will be called often. @Fberbari what are your thoughts?

Reply:
The poll for conversion function to poll the adc should work with a lower timeout, what would be a good value? 10-20 ms? I'll test to find the lowest value and update it.